File: Learn/30-Days-Of-Code/Day 26/nested.py
def fine_cost(rD, rM, rY, dD, dM, dY):
    if (rY < dY) or (rY <= dY and rM <= dM and rD <= dD):
        return 0
    elif rY == dY and rM == dM:
        return 15 * (rD - dD)
    elif rY == dY:
        return 500 * (rM - dM)
    else:
        return 10000

rD, rM, rY = [int(x) for x in input().split(' ')]
dD, dM, dY = [int(x) for x in input().split(' ')]
print(fine_cost(rD, rM, rY, dD, dM, dY))

File: Learn/30-Days-Of-Code/Day 26/nested.c
#include <stdio.h>

int fineCost(int returnDay, int returnMonth, int returnYear,
             int dueDay, int dueMonth, int dueYear) {
    if ((returnYear < dueYear) | (returnYear <= dueYear
            && returnMonth <= dueMonth \
            && returnDay <= dueDay)) {
        return 0;
    } else if (returnYear == dueYear && returnMonth == dueMonth) {
        return 15 * (returnDay - dueDay);
    } else if (returnYear == dueYear) {
        return 500 * (returnMonth - dueMonth);
    } else {
        return 10000;
    }
}

int main() {
    int rD, rM, rY;
    int dD, dM, dY;
    scanf("%d %d %d", &rD, &rM, &rY);
    scanf("%d %d %d", &dD, &dM, &dY);
    printf("%d\n", fineCost(rD, rM, rY, dD, dM, dY));
    return 0;
}
